pom.xml

```diff
@@ -269,9 +278,9 @@
     </profiles>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <felix.url>http://localhost:8080/system/console</felix.url>
+        <felix.url>http://localhost:4502/system/console</felix.url>
```

Comment:
this plugin is not dedicated to AEM :D is should work with even pure Felix / I was testing on it

Reply:
You wouldn't be so popular if it weren't fo those AEM developers but point taken :)